javascript_ts/2.echobot-with-counter/README.MD

```diff
@@ -1,15 +1,15 @@
-This sample shows how to create a simple Echo bot with state. The bot keeps simple counter step that increased with each user message. This bot example uses [`restify`](https://www.npmjs.com/package/restify). 
+This sample shows how to create a simple echo bot with state. The bot keeps simple counter step that increases with each user message. This bot example uses [`restify`](https://www.npmjs.com/package/restify). 
```

Comment:
Sorry I'm coming to this late, but can you change the second sentence to sound a little more like the following?

> The bot keeps _a simple step counter_ that increases with each user message.

The original sentence doesn't sound great when spoken aloud. And while I think I understand what is being conveyed, what do we mean by **_step_**?

Reply:
I rewrote this sentence slightly to "The bot maintains a simple counter that increases with each message from the user."